pkg/collector/python/init.go

```diff
@@ -414,7 +414,8 @@ func Initialize(paths ...string) error {
 		return err
 	}
 
-	if pkgconfigsetup.Datadog().GetBool("telemetry.enabled") && pkgconfigsetup.Datadog().GetBool("telemetry.python_memory") {
+	externalTelemetryEnabled := pkgconfigsetup.Datadog().GetBool("telemetry.enabled") || pkgconfigsetup.Datadog().GetBool("agent_telemetry.enabled")
+	if externalTelemetryEnabled && pkgconfigsetup.Datadog().GetBool("telemetry.python_memory") {
```

Comment:
I'm not sure I'm comfortable with this being enabled by default everywhere, as it is not risk free and can slow down python integrations.

But at the very least this needs a dedicated PR with a dedicated change log entry.

Reply:
I agree with that. We should slit the aggregation of metrics and the enablement of this feature for Python into two separate PRs

Reply:
Ok, will do.